pkgs/development/scheme-modules/scheme-bytestructures/default.nix

```diff
@@ -17,18 +17,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Wvs288K8BVjUuWvvzpDGBwOxL7mAXjVtgIwJAsQd0L4=";
   };
 
-  postConfigure = ''
-    sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
-    sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
-  '';
-
   nativeBuildInputs = [
     autoreconfHook pkg-config
   ];
   buildInputs = [
     guile
   ];
 
+  doCheck = true;
+  makeFlags = [ "GUILE_AUTO_COMPILE=0" ];
```

Comment:
I couldn't find why we want to set this or what this does.

Reply:
This is addressed at a review with similar changes at #189421 ([comment link](https://github.com/NixOS/nixpkgs/pull/189421#discussion_r962424917)).